sdk/identity/Azure.Identity.BrokeredAuthentication/src/Azure.Identity.BrokeredAuthentication.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is the implementation of the Azure SDK Client Library for Azure Identity</Description>
+    <AssemblyTitle>Microsoft Azure.Identity.BrokeredAuthentication Component</AssemblyTitle>
+    <Version>1.0.0-beta.1</Version>
+    <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
+    <!-- <ApiCompatVersion>1.0.0</ApiCompatVersion> -->
+    <PackageTags>Microsoft Azure Identity Broker;$(PackageCommonTags)</PackageTags>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net461</TargetFrameworks>
+    <NoWarn>$(NoWarn);3021</NoWarn>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Azure.Core" />
+    <!--<PackageReference Include="Azure.Identity" />-->
+    <!-- This should be changed back to a package reference once the required changes in Azure.Identity have shipped in the next GA -->
```

Comment:
Should we create an issue to track this?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/26879